README.md

```diff
@@ -0,0 +1,18 @@
+# Lazy
+
+A library for lazy objects, started life as a coredata/contrib package.
```

Comment:
I copied over all the existing documentation... which was none :P
What I wrote here is all I know about this library, without having to dig around the codebase.